### DISCUSSION TITLE
Github Student Pack

### DISCUSSION THREAD

**Discussion by pokhrelankit18** [community/community#86063] [GitHub Education]
### Select Topic Area

General

### Body

My Student ID verification has been in the Processing stage for so long? How long will this going to take?
![Screenshot 2024-01-09 234408](https://github.com/community/community/assets/154960622/00a373df-5f5e-4aa1-a434-3f67d4103f3a)

**Comment by ghostinhershell** [comment]
Hey there!  👋

Thanks for posting in the GitHub Community, @pokhrelankit18! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!
